### DISCUSSION TITLE
Add support for EdDSA

### DISCUSSION THREAD

**Discussion by ghost** [OpenIDC/mod_auth_openidc#983] [Ideas]
It seems currently only the following algorithms are supported:

```
#OIDCIDTokenSignedResponseAlg [RS256|RS384|RS512|PS256|PS384|PS512|HS256|HS384|HS512|ES256|ES384|ES512]
```

Would be nice to add `EdDSA` as well, as documented in "CFRG Elliptic Curve Diffie-Hellman (ECDH) and Signatures in JSON Object Signing and Encryption (JOSE)"

See [RFC 8037](https://www.rfc-editor.org/rfc/rfc8037)

**Comment by zandbelt** [comment]
Hi Francois! EdDSA support is going to be a long shot since mod_auth_openidc depends on [cjose](https://github.com/cisco/cjose) for JOSE capabilities and new developments there seems to have stopped. Hopefully someone will create a PR there someday...

**Reply by ghost** [reply]
Hi Hans,

Thanks for your reply! Ah, that is is unfortunate... 

Seems you are the new upstream now for cjose now though ;-)

https://packages.debian.org/bookworm/libcjose-dev
